launch/trigger_node.launch.xml

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0"?>
 <launch>
   <arg name="destination_ip" default="192.168.255.255" doc="The ip address of the camera/s to be triggered" />
```

Comment:
Is the destination_ip default to empty?

Reply:
`192.168.255.255` is on purpose, as it will broadcast on the 192.168.X.X subnet, so it will basically always work for any camera with an IP of 192.168.X.X. I suppose this could also be 255.255.255.255 which would be even more likely to work, but it mostly just serves as an example that you don't necessarily have to specify the exact IP of the camera (in case you have multiple cameras).

Reply:
In ros2 there is Python launch file, so this XML should be removed

Reply:
@Souphis, ROS2 now supports python, XML, and YAML launch files as of Foxy. I prefer XML since it's much less verbose and everyone is already familiar with them. See here for more details: https://docs.ros.org/en/foxy/How-To-Guides/Launch-file-different-formats.html